### DISCUSSION TITLE
AFE digital open/close input control

### DISCUSSION THREAD

**Discussion by Kabron287** [fhdm-dev/scoppy#145] [Scoppy Front end designs]
Why digital open/closed input control not done from the app?

**Comment by fhdm-dev** [comment]
Hi,

There's no particular reason except that I didn't need it for my AFEs. I just use pin headers and connect/disconnect them with jumpers to switch from DC to AC coupling.

But I can see that it could be useful.
Maybe I'll add it to future version of the app :)

**Comment by Kabron287** [comment]
Ideologically, this function is one of the main functions for an oscilloscope
